sc2bat/griffin_flight
# Feature/search flight

#63 #64 
- flight search result test done
- search flight done
- create flight dto tojson
- modify flight mapper
- create flight result mapper
- create airport repository get airport one
- modify flight model class name, flie name
- sign repository timeout check
- modify flight result model add airplane id
- modify flight result screen class name && file name
- books screen mock data add airplane id
- create common.dart calculate function
